modules/create_game.rb

```diff
@@ -0,0 +1,14 @@
+module GameCreator
+  def add_game
+    puts 'Create a new game'
+    print 'Is this game multiplayer(Y/N): '
+    multiplayer = gets.chomp
+    multiplayer = %w[y Y].include?(multiplayer)
+    print 'Enter last date the game was played [Enter date in format (yyyy-mm-dd)]: '
+    last_played_at = gets.chomp
+    print 'Enter the date of publish [Enter date in format (yyyy-mm-dd)]: '
+    publish_date = gets.chomp
+    @games << Game.new(multiplayer, last_played_at, publish_date)
```

Comment:
- [ ] **Required:** Great job implementing the console interactivity. 👏 However, the wrong date format crashes the app. You need a way to safeguard the app from crashing by either
- request from the user to enter the correct date or 
- returning them back to a safe place in the app.